helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -475,5 +471,10 @@ protected Map<String, ResourceAssignment> computeBestPossibleAssignment(
       return getBestPossibleAssignment(getAssignmentMetadataStore(), currentStateOutput,
           resourceMap.keySet());
     }
+
+    @Override
```

Comment:
Do we really need this? I think it will call super.close by default even you don't have this declaration here.

Reply:
Yeah, it should get from inheritance. I added it for better clarity. I can remove it

Reply:
I would suggest less code if possible : )

Reply:
Sounds good, it has been updated. Thanks for the review!